lava_test_plans/projects/lkft-android/cts-lkft-no-bionic-libcore.yaml

```diff
@@ -0,0 +1,25 @@
+{% extends "testcases/android-cts.yaml" %}
+
+{% set test_timeout = 480 %}
+{% set test_name = "cts-lkft" %}
+{% set cts_test_plan = "cts-lkft" %}
+
+{% block xts_test_params %}cts
+                --include-filter CtsAslrMallocTestCases
+                --include-filter CtsBluetoothTestCases
+                --include-filter CtsCameraTestCases
+                --include-filter CtsDisplayTestCases
+                --include-filter CtsDramTestCases
+                --include-filter CtsDrmTestCases
+                --include-filter CtsGraphicsTestCases
+                --include-filter CtsHardwareTestCases
+                --include-filter CtsJankDeviceTestCases
+                --include-filter CtsJniTestCases
+                --include-filter CtsLibcoreLegacy22TestCases
+                --include-filter CtsMonkeyTestCases
+                --include-filter CtsOsTestCases
+                --include-filter CtsSystemUiTestCases
+                --include-filter CtsSystemUiRenderingTestCases
+                --include-filter CtsUsbTests
+                --disable-reboot
+{% endblock xts_test_params %}
```

Comment:
I see this as a different android-cts-no-bionic-libcore.yaml test in the testcases directory.
Same with the others.
And not in the projects/lkft-android/ dir.

Reply:
there will be more workaround files like this to be added later, they are just specific to the lkft-android project or even specific to some android version of some device like the x15 master version here. it's better to put such files under the lkft-android project.  otherwise it will make the testcases directory messy with many such one case specific files there.